### PR TITLE
Update Action.js

### PR DESCRIPTION
**Pull request by Jaivardhan1**
Options for user to be determined by whoever is creating the scenario. Therefore they have to make up their own specific options.

### CODE REVIEW

**Review by benjaminxu251 — APPROVED** [review]

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/scenario_components/Action.js [hunk 1]
```diff
@@ -1,16 +1,88 @@
...
+  Button,
+  TextField,
+  Typography,
+  Container,
+  Box,
+} from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
 import Introduction from '../Introduction';
 import Title from '../Title';
+import Copyright from '../components/Copyright';
```
Comment (Enoch-Hsiao) [inline Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/scenario_components/Action.js:12]: Is it not import Copyright from '../Copyright';

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/scenario_components/Action.js [hunk 2]
```diff
@@ -1,16 +1,88 @@
...
+          <TextField
+            variant="outlined"
+            margin="normal"
+            required
+            fullWidth
+            id="option 1"
+            label="Input Option 1 Text"
+            name="option 1"
+            autoFocus
+          />
```
Comment (Enoch-Hsiao) [inline Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/scenario_components/Action.js:60]: Get rid of autoFocus, page scrolls down to option 1

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/scenario_components/Action.js (hunk 1 above)
Comment (Jaivardhan1) [reply Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/scenario_components/Action.js:12]: Changed

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/scenario_components/Action.js (hunk 2 above)
Comment (Jaivardhan1) [reply Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/scenario_components/Action.js:60]: Corrected

**Pull request merged**